apps/hash-frontend/src/pages/settings/organizations/new/index.page/create-org-form.tsx

```diff
@@ -28,6 +30,7 @@ export const CreateOrgForm = () => {
     }
 
     await refetchUser();
+    void refetchOrgs();
```

Comment:
Don't think it makes a huge difference, but I don't think there's a reason `refetchOrgs` has to happen after `refetchUser`:
```suggestion
    await Promise.all([refetchUser, refetchOrgs]);
```
Or if we don't want the redirect to be blocked:
```suggestion
    void refetchUser();
    void refetchOrgs();
```

Reply:
Thanks, done in https://github.com/hashintel/hash/pull/3191/commits/ecc9450d89b7cfe08ef8497ad9db9649a4c56967

Since we just fetch the org on that page we don't rely on the user/full orgs list